runtime/src/lib.rs

```diff
@@ -625,7 +625,7 @@ parameter_types! {
     pub const SubtensorInitialSenateRequiredStakePercentage: u64 = 1; // 1 percent of total stake
     pub const SubtensorInitialNetworkImmunity: u64 = 7 * 7200;
     pub const SubtensorInitialMinAllowedUids: u16 = 128;
-    pub const SubtensorInitialMinLockCost: u64 = 1_000_000_000_000; // 1000 TAO
+    pub const SubtensorInitialMinLockCost: u64 = 100_000_000_000; // 100 TAO
```

Comment:
This will be a problem for future runtime upgrades, we need to have two of these definitions with their own compiler flags to only insert them under specific conditions.

Reply:
Ah, makes sense. I can revert if that's simples for now. With the `BT_DEFAULT_TOKEN_WALLET` env var, this becomes more convenience.

Reply:
Reverted